charts/opa/templates/_helpers.tpl

```diff
@@ -107,3 +107,14 @@ admissionregistration.k8s.io/v1beta1
 {{- $tag := .Values.mgmt.image.tag | default .Chart.AppVersion -}}
 {{ printf "%s:%s" .Values.mgmt.image.repository $tag }}
 {{- end -}}
+
+{{/*
+Define default DNS policy
```

Comment:
This comment seems redundant, given how the code below says pretty much the same thing :)

Reply:
Removed. PTAL